packages/components/src/List/List.tsx

```diff
@@ -162,12 +165,17 @@ export const ListInternal = forwardRef(
 const ListStyle = styled.ul.withConfig({ shouldForwardProp })<
   HeightProps & WidthProps
 >`
+  ${fontFamily}
   ${reset}
   ${height}
   ${width}
 
   list-style: none;
   overflow: auto;
+
+  button {
+    ${fontFamily}
+  }
```

Comment:
I wonder if this selector is actually too narrow in this case? It seems like users could potentially render any form element in a list which might not inherit the type settings as expected. Maybe it makes sense to use the `*` selector instead?

```suggestion
  * {
    font-family: inherit;
  }
```

What do you think?

Reply:
This shouldn't be needed if the branch is off the latest `main` - `ListItem` _should_ just inherit the `List` font-family correctly.

Reply:
I updated to use `*` instead of `button` great suggestion thanks. As far as removing the code all together I just updated to be in the same place as main but if I remove it the font does not get updated. Not sure if I'm missing something here.

Reply:
I'm not a big fan of having a component reach into another component like this - with the refactor of the `reset` behavior it _should_ be possible to simply leverage CSS inheritance to get this to work.

Checking out branch locally to see if I can understand why it doesn't work without the specific selector here.

Reply:
I updated to use * instead of button great suggestion thanks. As far as removing the code all together I just updated to be in the same place as main but if I remove it the font does not get updated. Not sure if I'm missing something here.